### DISCUSSION TITLE
2/24 Hour format for the NSPAnel display

### DISCUSSION THREAD

**Discussion by abidullah** [Blackymas/NSPanel_HA_Blueprint#428] [Show and tell]
Just a modification to the original code which helps users to choose 12 or 24 format for their Nspanel Device from the Blueprint Automation.

<img width="1237" alt="Screen Shot 2023-03-13 at 5 15 36 AM" src="https://user-images.githubusercontent.com/2981996/224587099-865f9b7f-9f76-4145-9d75-7cc33bd0108e.png">

**Comment by Blackymas** [comment]
Can you create a pull request

**Comment by edwardtfn** [comment]
Done!
#510 
😉
